internal/errs2/ignore.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package errs2
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/zeebo/errs"
+	"google.golang.org/grpc"
+)
+
+// IgnoreCanceled returns nil, when the operation was about canceling.
+func IgnoreCanceled(originalError error) error {
+	err := originalError
+	for err != nil {
+		if err == context.Canceled ||
+			err == grpc.ErrServerStopped ||
+			err == http.ErrServerClosed {
+			return nil
+		}
+		unwrapped := errs.Unwrap(err)
+		if unwrapped == err {
```

Comment:
what does this case mean?

Reply:
It is a sanity check to avoid infinite loop when `errs.Unwrap` returns the `err` itself.